package.json

```diff
@@ -177,6 +176,7 @@
     "react-table": "^7.8.0",
     "react-virtualized-auto-sizer": "^1.0.5",
     "react-window": "^1.8.7",
+    "readline-sync": "^1.4.10",
```

Comment:
looks like this is only used by the foxy cli so maybe we should make it a dev dep

Reply:
Ooh, right you are.

Reply:
Moved to dev dep in https://github.com/shapeshift/web/pull/4506/commits/326089e07d24e7c56a811f20a889b99be40a6d44